version.txt

```diff
@@ -0,0 +1 @@
+97.0.0-beta.1
```

Comment:
We probably shouldn't add this here.

Reply:
Can you clarify what you mean? The version number is needed here for both shipit and the version-bump kind (basically same as what Fenix has).

Reply:
Right but neither of those things happen off the `main` branch, so I thought the file could be created at the branch point.  It's probably fine to have it on main like on fenix, in which case maybe bump to 98 though?

Reply:
Probably better to be consistent and mirror Fenix, but yeah, I'll update it to 98 now.